### PR TITLE
New resources added on NodeJS, MongoDB, Docker, Vim, TypeScript, GoLang

### PR DESCRIPTION
**Pull request by leJad**
## What type of PR is this? (check all applicable)

- [ ] Added new link to category
- [ ] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [x] Documentation update
- [ ] Frontend update

## Description
Added resources I use to learn NodeJS, MongoDB, Docker and others

## [optional] QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

### CODE REVIEW

**Comment by damla** [comment]
Hi @leJab,

Firstly, thanks for sharing these wonderful resources with us! Can you please fill the PR template we have you in the description?

**Comment by leJad** [comment]
> Hi @leJad ,
> 
> Firstly, thanks for sharing these wonderful resources with us! Can you please fill the PR template we gave you in the description?

Did it

**Comment by TheYkk** [comment]
lgtm

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (damla) [inline .github/CODEOWNERS:1]:
```suggestion
```

**Review by damla — DISMISSED** [review]
🚢🚢

**Comment by TheYkk** [comment]
lgtm

**Review by damla — APPROVED** [review]
🚢🚢

**Pull request merged**